src/template-tags/tickets.php

```diff
@@ -517,19 +518,40 @@ function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $
 			$sold_label = sprintf( _x( "%s'd going", 'RSVPs going', 'event-tickets' ), tribe_get_rsvp_label_singular() );
 		}
 
-		// Message for how many remain available
+		// Message for how many remain available.
 		if ( -1 === $available ) {
-			$status_counts[] = sprintf( _x( '%1$s available', 'unlimited remaining stock message', 'event-tickets' ), tribe_tickets_get_readable_amount( $available, $global_stock ) );
+			$status_counts[] = sprintf(
+				_x(
+					'%1$s available',
+					'unlimited remaining stock message',
+					'event-tickets'
+				),
+				tribe_tickets_get_readable_amount( $available, $global_stock )
+			);
 		} elseif ( $is_global ) {
-			$status_counts[] = sprintf( _x( '%1$d available of shared capacity', 'ticket shared capacity message (remaining stock)', 'event-tickets' ), tribe_tickets_get_readable_amount( $available ) );
+			$status_counts[] = sprintf(
+				_x(
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 77084ff3ab639750d5689bcb70c79227</sub>